eos-update-flatpak-repos

```diff
@@ -1232,7 +1232,8 @@ def rewrite_app_id(repo, mtree, old_id, new_id):
     # folder from the pre-migration deploy folder. the migration function
     # examines the commit metadata of the app being migrated to determine
     # whether the re-deploy has to carry out this extra step.
-    if metadata.remove_group(FLATPAK_METADATA_GROUP_EXTRA_DATA):
+    if metadata.has_group(FLATPAK_METADATA_GROUP_EXTRA_DATA):
+        metadata.remove_group(FLATPAK_METADATA_GROUP_EXTRA_DATA)
```

Comment:
Wouldn’t it be slightly better to catch and `pass` the exception? This is fine though.

Reply:
How's that? I always like a good excuse to use `try: except: else:`.

Reply:
:ok_hand: